fault-monitor/operational-status-monitor.hpp

```diff
@@ -91,6 +91,18 @@ class Monitor
      */
     void updateAssertedProperty(const std::vector<std::string>& ledGroupPaths,
                                 bool value);
+
+    /**
+     * @brief API to remove chassis critical association on the d-bus object
+     * This method removes the critical association between the given object
```

Comment:
brief should be enough, explanation not required.

Reply:
but why can't we have the explanation? it helps in understanding the purpose of a new api.